install-dev/langs/mx/language.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language>
+	<name><![CDATA[Español Mexicano (Mexican Spanish)]]></name>
```

Comment:
Can I add a commit to correct the line? It should be _Español MX (Spanish)_ if I refer to https://github.com/PrestaShop/PrestaShop/blob/develop/app/Resources/all_languages.json.

Reply:
Of course, you can improve your Pull Request until it's merged :)